### PR TITLE
Enigma2 media player

### PR DESCRIPTION
**Pull request by KavajNaruj**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#6977

### CODE REVIEW

File: source/_components/media_player.enigma2.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Volumio Media Player"
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: Needs an update.

File: source/_components/media_player.enigma2.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+description: "How to set up the Enigma2 media player platform"
+date: 2017-04-07
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: If there is no logo, please leave it out. The Home Assistant logo only be uses for Home Assistant related components/platforms.

File: source/_components/media_player.enigma2.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ha_category: Media Player
+ha_release: 0.42
+---
+
+The `Enigma2` platform allows you to control [Enigma2](http://wiki.blue-panel.com/index.php/Enigma2_(en)) linux based settopboxes through web API
+from Home Assistant.
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: There is no need to add line breaks. The renderer will take care about this for you.

File: source/_components/media_player.enigma2.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+title: "Volumio Media Player"
+description: "How to set up the Enigma2 media player platform"
+date: 2017-04-07
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Media Player
+ha_release: 0.42
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: Will be 0.43. As 0.42 will be released in a couple of hours.

File: source/_components/media_player.enigma2.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+The `Enigma2` platform allows you to control [Enigma2](http://wiki.blue-panel.com/index.php/Enigma2_(en)) linux based settopboxes through web API
+from Home Assistant.
+
+To test if your web api works: Navigate your browser at http://192.168.1.x/web/powerstate, you should see there an XML output.
+x - put ip address of your box
+
+To add Enigma2 player to your installation, add the following to
+your `configuration.yaml` file.
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: Dito.

File: source/_components/media_player.enigma2.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: enigma
+    host: 192.168.1.x
+    port: 80
+    username: root
+    password: pass
```
Comment (fabaff) [inline source/_components/media_player.enigma2.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Comment by balloob** [comment]
Closing because PR in main repo got closed.

**Pull request closed without merging**